src/app/calculator/motors/motor-drive/motor-drive-table/motor-drive-table.component.ts

```diff
@@ -11,14 +11,18 @@ export class MotorDriveTableComponent implements OnInit {
   results: MotorDriveOutputs;
 
   @ViewChild('copyTable', { static: false }) copyTable: ElementRef;
-  tableString: any;
+  
   @ViewChild('copyTable2', { static: false }) copyTable2: ElementRef;
+
+  tableString: any;
```

Comment:
This existed before your changes, but here and any other files you touched where tableString (or any other property) use any - change type to string or whatever the type should be.